src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_logging.v2beta1.json.baseline

```diff
@@ -24,24 +32,29 @@ google.auth.getApplicationDefault(function(err, authClient) {
   }
 
   var request = {
-    // TODO: Change placeholders below to appropriate parameter values for the 'list' method:
 
-    resource: {},
+    resource: {
+      // TODO: Add the desired properties.
+    },
 
-    // Auth client
     auth: authClient
   };
 
-
-  var recur = function(err, result) {
+  var recur = function(err, response) {
```

Comment:
I think you missed the function name:

```
foo(function handlePage(err, response) {
  foo(handlePage);
});
```

but yes, I agree (I have JS readability).